pages/linux/pivpn.md

```diff
@@ -0,0 +1,29 @@
+# pivpn
+
+> Easy security-hardened OpenVPN setup and manager.
+> Originally designed for the Raspberry Pi, but works on other Linux devices too.
```

Comment:
I wonder if this line is necessary ?

Reply:
I think so. `Pi` is in the name, which is implies that it only work on Raspberry Pis.